main/updater.js

```diff
@@ -31,7 +41,13 @@ function setup (/** @type {import('./typings').Context} */ _ctx) {
   require('electron').autoUpdater.on('before-quit-for-update', beforeQuitCleanup)
 }
 
-module.exports = async function (/** @type {import('./typings').Context} */ ctx) {
+module.exports = async function setupUpdater (/** @type {import('./typings').Context} */ ctx) {
+  ctx.getUpdaterStatus = async function getUpdaterStatus () {
```

Comment:
Does this function need to be `async`?

Reply:
@bajtos

Reply:
I was thinking about making the function future-proof. I am fine with changing it to a regular (sync) function.

Reply:
Fixed in 6dffefc